tests/shared/src/test/scala/io/iteratee/EitherTTests.scala

```diff
@@ -0,0 +1,107 @@
+package io.iteratee
+
+import cats.{ Eval, Id, MonadError }
+import cats.arrow.FunctionK
+import cats.data.EitherT
+import io.iteratee.tests.{ EnumerateeSuite, StackSafeEnumeratorSuite, IterateeErrorSuite, EitherTSuite, eqThrowable }
+import org.scalacheck.Arbitrary
+
+class EitherTEnumerateeTests extends EnumerateeSuite[({ type L[x] = EitherT[Eval, Throwable, x] })#L]
+    with EitherTSuite {
+  "take" should "work with more than Int.MaxValue values" in forAll { (n: Int) =>
```

Comment:
this test seems to have nothing to do with Either. Shame to have to have it here.